cluster/gce/gci/configure-helper.sh

```diff
@@ -737,6 +737,10 @@ function start-kube-apiserver {
     params+=" --enable-garbage-collector=${ENABLE_GARBAGE_COLLECTOR}"
   fi
   if [[ -n "${NUM_NODES:-}" ]]; then
+    # If the cluster is large, increase max-requests-inflight limit in apiserver.
+    if [[ "${NUM_NODES}" -ge 1000 ]]; then
+      params+=" --max-requests-inflight=1500"
```

Comment:
I'm wondering if we shouldn't check the current value somehow, in case that someone/thing will set the default higher...